SF iOS/SF iOS/EventDetails/EventDetailsViewController.m

```diff
@@ -115,10 +114,16 @@ - (void)viewDidLoad {
     
     [self.view addSubview:self.containerStack];
     // Extend under status bar
-    [self.containerStack.topAnchor constraintEqualToAnchor:self.view.topAnchor].active = true;
-    [self.containerStack.leftAnchor constraintEqualToAnchor:self.view.leftAnchor].active = true;
-    [self.containerStack.bottomAnchor constraintEqualToAnchor:self.view.bottomAnchor].active = true;
-    [self.containerStack.rightAnchor constraintEqualToAnchor:self.view.rightAnchor].active = true;
+    [NSLayoutConstraint activateConstraints:
+     @[
+       [self.travelTimesView.heightAnchor constraintGreaterThanOrEqualToConstant:141],
+
+       [self.containerStack.topAnchor constraintEqualToAnchor:self.view.topAnchor],
```

Comment:
Not a big deal but did you put the space here intentionally?

Reply:
I see down below. This is consistent with what you did since it’s sort of a didfeeent group of constraints. Reasonable.